### ISSUE TITLE
Example projects: Failed to download tarball for `pax-component-library` at version 0.15.5

### ISSUE THREAD

**Issue by eliot-akira** [paxengine/pax#208]
Following the [instructions to run an example project](https://github.com/paxengine/pax?tab=readme-ov-file#examples), I'm seeing an error:

```
$ cd examples/src/space-game && pax-cli run --target=web

Failed to download tarball for pax-component-library at version 0.15.5. Status: 403 Forbidden
```

Same in other example projects. I'm guessing it shouldn't be fetching a tarball since this module is within the monorepo.

Searching in the codebase, `pax-component-library` is the only module whose version is at `0.15.1` in `Cargo.toml`. All other modules are at `0.15.5`. This looks related to the issue.

I tried updating its version number and re-running `pax-cli run`, but the fetch error is still there. Thinking maybe the non-existing external dependency is cached, I removed `Cargo.lock` but still won't build.

**Comment by mxmind** [comment]
I faced the same issue

**Comment by zackbrown** [comment]
Sorry about this and thanks for the report, @eliot-akira and @mxmind . Looks like we botched a release by not publishing `pax-component-library` to crates.io.  We will fix this with our next release and will update here when we do.

In the meantime if you use the `./pax` shell script in the examples directories instead of `pax-cli`, you should be able to work around this.  (e.g. `cd examples/src/fireworks && ./pax run`)  This helper script will use local filesystem crates instead of pulling from crates.io.

**Comment by eliot-akira** [comment]
Thank you, I confirm that `./pax` solves it. The compilation completes successfully and it starts the Pax Web server.

**Comment by zackbrown** [comment]
Just published a fairly substantial update (#213, released as 0.21.0)

Based on my testing of a fresh clone with both `./pax run` and `pax-cli run` (after installing latest pax-cli with `cargo install --force pax-cli`), both work as expected.

Marking this issue resolved.  Thanks again for the report.
